src/langchain-agent-auth.mdx

```diff
@@ -0,0 +1,115 @@
+---
+title: LangChain Agent Auth
+---
+LangChain Agent Auth enables secure access from agents to any other system via OAuth 2.0 credentials. With LangChain Agent Auth, build your agents to securely connect to any system on a person's behalf.
+
+## Overview
+
+LangChain Agent Auth is designed to solve the challenge of secure authentication for autonomous agents. Instead of embedding credentials directly in agent code or requiring manual intervention for each API call, agents can use the OAuth flow to obtain time-limited access tokens for external services.
+
+The system consists of:
+- **Authentication Server**: Manages OAuth flows and stores credentials securely
+- **Client SDK**: Python library for easy integration with agents
+
+## Installation
+
+Install directly from the wheel file:
+
+For use with LangGraph:
+```bash
+pip install https://docs.langchain.com/langchain_auth-0.1.0-py3-none-any.whl[langgraph]
```

Comment:
this will change once we host the package on PyPi in our typical way, but for now any alpha user can download the package from our docs site (where it is less visible to the public than PyPi)